src/utils/sortPeople.ts

```diff
@@ -0,0 +1,46 @@
+import { Person } from '../types';
+import { SortBy, SortOrder } from '../types/Sort';
+
+export function sortPeople(
+  people: Person[],
+  field: keyof Person,
+  order: string,
```

Comment:
The `order` parameter should be of type `SortOrder` instead of `string`. This will ensure that only the values defined in the `SortOrder` enum can be passed as arguments, improving type safety.